subsys/mgmt/mcumgr/Kconfig

```diff
@@ -32,6 +32,25 @@ config MCUMGR_SMP_LEGACY_RC_BEHAVIOUR
 	  response will be empty with the new behaviour enabled, as opposed to
 	  the old behaviour where the rc field will be present in the response.
 
+menuconfig SMP_CLIENT
+	bool "SMP Client support"
+	help
+	  This will enable SMP Request generation and response handling.
+if SMP_CLIENT
+config SMP_CMD_DEFAULT_LIFE_TIME
+	int "SMP command Lifetime in seconds"
+	range 2 30
+	default 5
+	help
+	  This define how many times Packet is re-sended by using SMP_CMD_RETRY_TIME value.
+config SMP_CMD_RETRY_TIME
+	int "SMP command re-send periond in ms"
+	range 100 1000
+	default 500
+	help
+	  This will set how long time SMP client wait response before re-send
```

Comment:
This values do not make sense, an erase can take upwards of 10 seconds, so why would you resend the command after 0.5s?